data-intake/ddi/README.md

```diff
@@ -0,0 +1,16 @@
+# Direct Data Intake API
+The Direct Data Intake API is suitable for Visier customers who have, at least in part, data to load into Visier that require no additional transformations. This is what makes this process 'direct'. In fact, the process is so streamlined that the column names in the source data files have match the names of the properties to load in the target objects.
```

Comment:
> data files have match the names of

Did you mean `have matched` or like `matches automatically` here?

Reply:
`have to match`. Thanks. Will fix